src/utils/forks.tsx

```diff
@@ -0,0 +1,70 @@
+import { getReposInOrgGQL } from 'bot/graphql'
+import { personalOctokit } from 'bot/octokit'
+import { useSession } from 'next-auth/react'
+import { useEffect, useState } from 'react'
+import { ForksObject } from 'types/forks'
+
+const getForksInOrg = async (accessToken: string, login: string) => {
+  const res = await personalOctokit(accessToken).graphql.paginate<ForksObject>(
+    getReposInOrgGQL,
+    {
+      login,
+      isFork: true,
+    },
+  )
+
+  return {
+    organization: {
+      repositories: {
+        totalCount: res.organization.repositories.totalCount,
+        nodes: res.organization.repositories.nodes.map((node) => ({
+          id: node.databaseId,
+          name: node.name,
+          isPrivate: node.isPrivate,
+          updatedAt: node.updatedAt,
+          owner: {
+            avatarUrl: node.owner.avatarUrl,
+            login: node.owner.login,
+          },
+          parent: {
+            name: node.parent?.name,
+            owner: {
+              login: node.parent?.owner.login,
+              avatarUrl: node.parent?.owner.avatarUrl,
+            },
+          },
+          languages: {
+            nodes: node.languages.nodes.map((node) => ({
+              name: node.name,
+              color: node.color as string,
+            })),
+          },
+          refs: {
+            totalCount: node.refs.totalCount,
+          },
+        })),
```

Comment:
```suggestion
        nodes: res.organization.repositories.nodes.map((node) => ({
          ...node,
          parent: {
            name: node.parent?.name,
            owner: {
              login: node.parent?.owner.login,
              avatarUrl: node.parent?.owner.avatarUrl,
            },
          },
          languages: {
            nodes: node.languages.nodes.map((node) => ({
              name: node.name,
              color: node.color as string,
            })),
          },
        })),
```

This may be easier to write 🤔